content/doc/book/installing/docker.adoc

```diff
@@ -20,6 +20,14 @@ changes against the procedures documented in the
 Jenkins Documentation.
 ////
 
+
+== Java Requirement
+
+Jenkins requires Java 17 or Java 21 (64-bit).  
+Install Java before proceeding.  
+Refer to the [_Installation Requirements_](./_installation_requirements.adoc) page for details.
+
+
```

Comment:
Java is already included in the container definition.  Declaring that Java 17 or Java 21 should be installed first does not make sense in this page because the container image already includes the correct Java version
```suggestion
```